lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb

```diff
@@ -109,6 +110,21 @@ def add_resources
             end
           end
 
+          def add_developer_files
+            UI.message '- Adding development pod helper files' do
+              file_accessors.each do |file_accessor|
+                pod_name = file_accessor.spec.name
+                next unless sandbox.local?(pod_name)
+                root_name = Specification.root_name(pod_name)
+                paths = file_accessor.developer_files
+                paths.each do |path|
+                  group = pods_project.group_for_spec(root_name, :developer)
+                  pods_project.add_file_reference(path, group, false)
+                end
```

Comment:
I had this as 

```ruby
add_file_accessors_paths_to_pods_group(:developer_files, :developer, true)
```

but that caused some duplicate groups when subspecs were specified. I really just needed `group_for_spec` to use the root name instead of `file_accessor.spec.name`